multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -36,14 +36,31 @@ coreos:
       command: start
       enable: true
       content: |
+{{ if isset .ContainerRuntime "rkt" }}
+        [Unit]
+        Requires=load-rkt-stage1.service
+        After=load-rkt-stage1.service
+{{ end }}
         [Service]
         Environment=KUBELET_VERSION={{.K8sVer}}
         Environment=KUBELET_ACI={{.HyperkubeImageRepo}}
-        Environment="RKT_OPTS=--volume dns,kind=host,source=/etc/resolv.conf --mount volume=dns,target=/etc/resolv.conf"
+        Environment="RKT_OPTS=--volume dns,kind=host,source=/etc/resolv.conf \
+          --mount volume=dns,target=/etc/resolv.conf \
+          --volume rktbin,kind=host,source=/usr/bin/rkt \
+          --mount volume=rktbin,target=/usr/bin/rkt \
```

Comment:
Also, just a thought, but, at one point we were looking for something called a `naked-stage1` such that the container wasn't in a chroot but rather just a prefix and could call host binaries or container binaries and things would just work. The main problem was sorting out a solution dependent libaries but also making sure the container was happy not living at root.

Given that old idea and the solution you came up with here, what if fly-stage1 just had the option (with a flag) to add binaries from the host to the containers executable path but have them execute with that nsenter command above? It means you have to specify your escape hatches but thats how fly works anyway.

Anyway just a thought, I'm going to prioritize getting the other blockers on this PR fixed and maybe do a followup with your solution to make rkt gc work.